rasterio/merge.py

```diff
@@ -45,6 +47,10 @@ def merge(datasets, bounds=None, res=None, nodata=None, precision=7, indexes=Non
         in the first input raster.
     indexes : list of ints or a single int, optional
         bands to read and merge
+    method : str or callable
+        pre-defined or custom merging method
```

Comment:
Can you add a list of the pre-defined names here?

Reply:
like this, or is this too verbose?